vaadin-date-picker-flow-parent/vaadin-date-picker-flow/src/main/resources/META-INF/resources/frontend/datepickerConnector.js

```diff
@@ -82,20 +86,24 @@ import { extractDateParts, parseDate as _parseDate } from '@vaadin/date-picker/s
 
             // Update century if the last parsed date is the same except the century.
             if (datepicker.$connector._lastParseStatus === 'successful') {
-              if (datepicker.$connector._lastParsedDate.day === date.getDate() &&
+              if (
+                datepicker.$connector._lastParsedDate.day === date.getDate() &&
```

Comment:
Note: some changes are caused by Prettier. I decided to commit them as they are quite small.